### PR TITLE
feat(alerts): add special tag frequency condition

### PR DESCRIPTION
**Pull request by JoshFerge**
Adds a new alert condition which will be added in https://github.com/getsentry/sentry/pull/78886. Gates it to only those users with the flag enabled.

### CODE REVIEW

File: static/app/views/alerts/utils/constants.tsx [hunk 1]
```diff
@@ -9,6 +10,8 @@ export const CHANGE_ALERT_PLACEHOLDERS_LABELS = {
     'Number of events in an issue is',
   'sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyCondition':
     'Number of users affected by an issue is',
+  'sentry.rules.conditions.event_frequency.EventUniqueUserFrequencyConditionWithConditions':
+    'Number of users affected by an issue (with tag values equal) is',
   'sentry.rules.conditions.event_frequency.EventFrequencyPercentCondition':
     'Percent of sessions affected by an issue is',
 };
```
Comment (evanpurkhiser) [inline static/app/views/alerts/utils/constants.tsx:17]: I guess we already don't translate these huh

File: static/app/views/alerts/rules/issue/index.tsx [hunk 2]
```diff
@@ -794,17 +794,32 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     if (!organization.features.includes('change-alerts')) {
       return this.state.configs?.conditions ?? null;
     }
+    let conditions = this.state.configs?.conditions ?? null;
 
-    return (
-      this.state.configs?.conditions?.map(condition =>
+    if (
```
Comment (JoshFerge) [inline static/app/views/alerts/rules/issue/index.tsx:799]: i'm wondering if i need to add this logic in here too? https://github.com/getsentry/sentry/blob/158c3c4bb13636fa8a775b2420c1fc1b412db5f5/static/app/views/alerts/rules/issue/ruleNodeList.tsx#L181 not sure where that's used

File: static/app/views/alerts/rules/issue/index.tsx [hunk 3]
```diff
@@ -794,17 +794,32 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     if (!organization.features.includes('change-alerts')) {
       return this.state.configs?.conditions ?? null;
     }
+    let conditions = this.state.configs?.conditions ?? null;
```
Comment (scttcper) [inline static/app/views/alerts/rules/issue/index.tsx:797]: i feel like we could check if its null earlier and return early save a bunch of the .? and ?? null repeats

**Review by scttcper — APPROVED** [review]

File: static/app/views/alerts/rules/issue/index.tsx (hunk 3 above)
Comment (JoshFerge) [reply static/app/views/alerts/rules/issue/index.tsx:797]: i just like writing lots of `???` :)

File: static/app/views/alerts/rules/issue/index.tsx (hunk 2 above)
Comment (scttcper) [reply static/app/views/alerts/rules/issue/index.tsx:799]: i think change alerts should be available to most orgs? maybe?

Comment (JoshFerge) [reply static/app/views/alerts/rules/issue/index.tsx:799]: no sorry, i mean should my new flag be applied in this other file too. in my local testing this correctly gated the condition, but i'm not sure where this other ruleNodeList thing is used

**Comment by JoshFerge** [comment]
reverting as this wasn't hiding the feature correctly in a different part of the alerts UI. will re-open with a fix for that.

**Pull request merged**